stable/dask/templates/NOTES.txt

```diff
@@ -9,16 +9,60 @@ This release includes a Dask scheduler, {{ .Values.worker.replicas }} Dask worke
 
 The Jupyter notebook server and Dask scheduler expose external services to
 which you can connect to manage notebooks, or connect directly to the Dask
-cluster.   You can get these addresses by running the following:
+cluster. You can get these addresses by running the following:
+
+{{- if contains "NodePort" .Values.scheduler.serviceType }}
+
+  export DASK_SCHEDULER=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath='{.items[0].status.addresses[0].address}')
+  export DASK_SCHEDULER_UI_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath='{.items[0].status.addresses[0].address}')
+  export DASK_SCHEDULER_PORT=$(kubectl get svc --namespace default {{ template "dask.fullname" . }}-scheduler -o jsonpath='{.spec.ports[?(@.name=="{{ template "dask.fullname" . }}-{{ .Values.scheduler.name }}")].nodePort}')
```

Comment:
This and several other lines should have "--namespace {{ .Release.Namespace }}" not "--namespace default" for when it's run in another namespace.

Reply:
Good catch, just addressed this in commit [below](https://github.com/helm/charts/pull/12552/commits/875b4f73af54c064a706d2c0335d7a20b48f5d71).